core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1915,8 +1915,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     )
 
     // todo: update this line as part of https://broadworkbench.atlassian.net/browse/CA-1220
-    policyGroupsToRoles.toList.traverse {case (email, roles) =>
-      googleIamDao.addIamRoles(GoogleProject(googleProject.value), email, MemberType.Group, roles)} // addIamRoles logic includes retries
+    policyGroupsToRoles.toList.foldLeft(Future(true)){case (result, (email, roles)) => googleIamDao.addIamRoles(GoogleProject(googleProject.value), email, MemberType.Group, roles)}
```

Comment:
I feel like there's gotta be a cleaner way to run all in serial, but I don't know the scala command if so.

Reply:
I think this is still going to run them in parallel. To run them in serial you want to chain the futures with `flatMap`, something like:
```
policyGroupsToRoles.toList.foldLeft(Future(true)) { case (result, (email, roles)) => 
  result.flatMap(_ => googleIamDao.addIamRoles(GoogleProject(googleProject.value), email, MemberType.Group, roles))
}
```

Reply:
Ah! Got it, thanks @rtitle!